honeybee_schema/energy/constructionset.py

```diff
@@ -35,6 +41,30 @@ class WallSetAbridged(NoExtraBaseModel):
     )
 
 
+class WallSet(WallSetAbridged):
+    """A set of constructions for wall assemblies."""
```

Comment:
This isn't how the non-abridged `ConstructionSet` schema works exactly. The non-abridged `ConstrucitonSet` still uses abridged sub-sets (`WallSetAbrdiged`, `ApertureSetAbrdiged`, etc.). In this sense, the `ConstructionSet` is exactly like the `ConstructionSetAbridged`. The only difference is that the `ConstructionSet` includes additional keys for `constructions` that list all of the Abridged constructions (`OpaqueConstructionAbridged`, `WindowConstructionAbridged`, etc.) that are referenced in the sub-sets. It also includes a `materials` key that lists all unique materials(`EnergyMaterial`, `EnergyWindowMaterialSimpleGlazSys`, etc.).